packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -4441,6 +4429,30 @@ export class ThinEngine {
         );
     }
 
+    /**
+     * Calls to the GL texImage2D and texImage3D functions require three arguments describing the pixel format of the texture.
+     * createTexture derives these from the format and useSRGBBuffer arguments and also the file extension of the URL it's working with.
+     * This function encapsulates that derivation for easy unit testing.
+     * @param format Babylon's format enum, as specified in ITextureCreationOptions.
+     * @param fileExtension The file extension including the dot, e.g. .jpg.
+     * @param useSRGBBuffer Use SRGB not linear.
+     * @returns The options to pass to texImage2D or texImage3D calls.
+     */

```

Comment:
You should add `@internal` as the method is not meant to be used by the end user.

Reply:
Done.